index.js

```diff
@@ -408,11 +408,16 @@ class NYC {
 
   async getCoverageMapFromAllCoverageFiles (baseDirectory) {
     const map = libCoverage.createCoverageMap({})
+    const files = await this.coverageFiles(baseDirectory)
 
-    const data = await this.coverageData(baseDirectory)
```

Comment:
The changes to this function seem fine, one request could you add a comment before the definition of `coverageData` saying `// TODO: Remove from nyc v16`?